datafusion/core/tests/sql/aggregates.rs

```diff
@@ -37,7 +37,7 @@ async fn csv_query_array_agg_distinct() -> Result<()> {
         Schema::new(vec![Field::new_list(
             "ARRAY_AGG(DISTINCT aggregate_test_100.c2)",
             Field::new("item", DataType::UInt32, true),
-            false
```

Comment:
Do we have access to the nullability of input arguments when defining UDAFs? I did not see how to maintain the behavior here.